plugins/lang/python/python_plugin_context.cc

```diff
@@ -199,6 +199,8 @@ class CatchOutErr:
         self.method=method
     def write(self, txt):
         self.catchOut.write(txt, self.method)
+    def flush(self):
```

Comment:
whoa, we can inject Python code in the context of every execution??? if so, we can bolt on all sorts of useful stuff to the mldb object right now :) /cc @mailletf ?